extensions/objects/templates/FilepathEditor.js

```diff
@@ -310,8 +312,13 @@ oppia.directive('filepathEditor', [
 
         var getTrustedResourceUrlForImageFileName = function(imageFileName) {
           var encodedFilepath = window.encodeURIComponent(imageFileName);
-          return $sce.trustAsResourceUrl(
-            '/imagehandler/' + $scope.explorationId + '/' + encodedFilepath);
+          var objectUrl = '';
+          AssetsBackendApiService.loadImage(
+            $scope.explorationId, imageFileName)
+            .then(function(loadedImageFile) {
+              objectUrl = URL.createObjectURL(loadedImageFile.data);
+            });
+          return objectUrl;
```

Comment:
Hi @vojtechjelinek , Here I return the objectUrl but there is no point in updating the objectUrl(after the image loads), if the function already returned.
Could this be a problem?
I can add a promise here, but that won't solve this I guess because later in the code this function (`getTrustedResourceUrlForImage`) is called in the function `setSavedImageFilename` for setting the value of the `savedImageUrl`.

Reply:
We can load the image directly from the url instead of loading it by loadImage, so can you create some helper method in AssetsBackendApiService or other service that would return the url for that image?

Reply:
Okay